source/docs/design-skills/cad.rst

```diff
@@ -18,6 +18,8 @@ So, what's the point of CAD, then? It is encouraged that new teams try their han
 #. CAD solves a lot of preventable headaches, such as spacing issues. Thus, it will save time when you discover problems in CAD that you can remediate before you build your robot.
 #. CAD is a professional tool that is used in many STEM fields. Having CAD knowledge and skills will be beneficial in your future career, should you study and work in those fields.
 #. If you desire to create 3D-printed or machined parts, CAD will be necessary to do so.
+#. CAD can reduce the cost of building a robot by helping to determine which parts you need for a robot before actually spending the money to buy those parts.
+#. CAD can also be a big time saver in the build process by giving you and your team something to go off of when building.
```

Comment:
```suggestion
#. CAD solves a lot of preventable headaches, such as spacing issues. Thus, it will save time when you discover problems in CAD that you can remediate before you build your robot.
#. CAD is a professional tool that is used in many STEM fields. Having CAD knowledge and skills will be beneficial in your future career, should you study and work in those fields.
#. If you desire to create 3D-printed or machined parts, CAD will be necessary to do so.
#. CAD can reduce the cost of building a robot by helping to determine which parts you need for a robot before actually spending the money to buy those parts.
```
CAD doesn't really give you an idea of what to go off of, it lets you experiment with that more easily—that's a design skill though, not a CAD skill. The point about preventable headaches covers this in my opinion.